test/liquid_handle_builder_test.py

```diff
@@ -122,6 +122,25 @@ def test_position_z_builder(self):
             structured_position_z
         )
 
+        extra_detection_arguments_positions_z = {
+            "reference": flat_position_z["reference"],
+            "offset": flat_position_z["offset"],
+            "move_rate": flat_position_z["move_rate"],
+            "detection": {
+                "method": flat_position_z["detection_method"],
+                "threshold": flat_position_z["detection_threshold"],
+                "duration": flat_position_z["detection_duration"],
+                "fallback": flat_position_z["detection_fallback"],
+                "x_edge_steepness": 20,
```

Comment:
nit: I'd prefer avoiding referencing specific `x_`params here and instead throwing some other field here that's obviously out of the ordinary (e.g. `foo`, `test` or similar).

This is partly because we don't want those specific fields names to seem important to this test and partly because we'd like to keep the notion of `x_`params out of autoprotocol python as much as possible.